leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -466,4 +557,26 @@ public void run() {
             }
         }
     }
+
+    // Class for holding OSCORE related information
+    private static class OSCoreSettings {
+        public String masterSecret;
+        public String masterSalt;
+        public String idContext;
```

Comment:
it sounds not to be used ?

Reply:
Yes, it is not used. It seems the OSCORE object as defined in LWM2M 1.1 does not support the ID Context parameter currently. This is something that the main OSCORE implementation uses and would be nice to have support for in LWM2M and Leshan long term. I have discussed this with Ericsson and we will see where things lead.

I removed this variable in commit 09a48e3d46c1b9234b84bf10df66f43c35a89fc1